src/Tools/dotnet-dsproxy/dotnet-dsproxy.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(BundleTools)' == 'true'">netcoreapp3.1</TargetFramework>
+    <TargetFramework Condition="'$(BundleTools)' != 'true'">netcoreapp2.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ToolCommandName>dotnet-dsproxy</ToolCommandName>
+    <RootNamespace>Microsoft.Diagnostics.Tools.DSProxy</RootNamespace>
+    <Description>.NET Performance Diagnostic Server Proxy Tool</Description>
+    <PackageTags>Diagnostic</PackageTags>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Common\CommandExtensions.cs" Link="CommandExtensions.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Microsoft.Diagnostics.NETCore.Client\Microsoft.Diagnostics.NETCore.Client.csproj" />
+  </ItemGroup>
+
```

Comment:
Should we mark this as non-shipping until we go through the e2e review process to not end up with this in nuget.org?

Reply:
Yes please 😄

Reply:
Yes

Reply:
How do we mark things as non-shipping? Another question/thought, is there any way to get the tool to customers testing out .net6 previews before e2e review process is complete? Tool currently includes a large warning on start that its an experimental tool and if you try to bind anything except loopback interface it will create some additional warnings informing end user that the tool is only a development/testing tool and should only be bound to loopback interface.

Reply:
```xml
  <PropertyGroup>
    <IsShipping>false</IsShipping>
  </PropertyGroup>
```

Reply:
> Another question/thought, is there any way to get the tool to customers testing out .net6 previews before e2e review process is complete?

This just prevents us from accidentally shipping to nuget.org. Depending on how motivated your customers are, you can always have them pull down the tool from our feed at `https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-tools/nuget/v3/index.json`

Reply:
I'm happy to collaborate with you if you want to write-up something for customers to try

Reply:
OK, so then users could do something similar to what we did with the dotnet-monitor:

`dotnet tool install -g dotnet-dsrouter --add-source https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-tools/nuget/v3/index.json --version 6.0.0-preview.*`

in order to use it during .net6 previews.

Reply:
Yes, that should work